drivers/include/at86rf2xx.h

```diff
@@ -61,23 +61,13 @@ extern "C" {
  */
 #ifdef MODULE_AT86RF212B
 /* the AT86RF212B has a sub-1GHz radio */
-#define AT86RF2XX_MIN_CHANNEL           (0)
-#define AT86RF2XX_MAX_CHANNEL           (10)
-#ifdef DEFAULT_CHANNEL
-#define AT86RF2XX_DEFAULT_CHANNEL (DEFAULT_CHANNEL)
-#endif
-#ifndef AT86RF2XX_DEFAULT_CHANNEL
-#define AT86RF2XX_DEFAULT_CHANNEL       (5)
-#endif
+#define AT86RF2XX_MIN_CHANNEL           (IEEE802154_CHANNEL_MIN_SUBGHZ)
+#define AT86RF2XX_MAX_CHANNEL           (IEEE802154_CHANNEL_MAX_SUBGHZ)
+#define AT86RF2XX_DEFAULT_CHANNEL       (IEEE802154_DEFAULT_SUBGHZ_CHANNEL)
 #else
-#define AT86RF2XX_MIN_CHANNEL           (11U)
-#define AT86RF2XX_MAX_CHANNEL           (26U)
-#ifdef DEFAULT_CHANNEL
-#define AT86RF2XX_DEFAULT_CHANNEL (DEFAULT_CHANNEL)
-#endif
-#ifndef AT86RF2XX_DEFAULT_CHANNEL
-#define AT86RF2XX_DEFAULT_CHANNEL       (26U)
-#endif
+#define AT86RF2XX_MIN_CHANNEL           (IEEE802154_CHANNEL_MIN)
+#define AT86RF2XX_MAX_CHANNEL           (IEEE802154_CHANNEL_MAX)
+#define AT86RF2XX_DEFAULT_CHANNEL       (IEEE802154_DEFAULT_CHANNEL)
```

Comment:
look at how they are grouped in `net/ieee802154.h` and this naming makes more sense ;-)